html4j/src/main/java/org/teavm/html4j/HTML4jResourceSupplier.java

```diff
@@ -27,17 +26,10 @@
 public class HTML4jResourceSupplier implements ResourceSupplier {
     @Override
     public String[] supplyResources(ClassLoader classLoader, ListableClassReaderSource classSource) {
-        Set<String> resources = new HashSet<>();
-        for (String className : classSource.getClassNames()) {
-            final int lastDot = className.lastIndexOf('.');
-            if (lastDot == -1) {
-                continue;
-            }
-            String packageName = className.substring(0, lastDot);
-            String resourceName = packageName.replace('.', '/') + "/" + "jvm.txt";
-            resources.add(resourceName);
+        String resources = System.getProperty("resourcesSupplier");
```

Comment:
System property isn't the best way to propagate information between Mojo and the supplier, but it was the easiest. There is a quite a lot of TeaVM layers in between and I am not sure what is the best way to cross through them.